src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -927,6 +927,45 @@ BOOST_AUTO_TEST_CASE(SimpleLogSoftmaxLayerTest)
 }
 
 /*
+ * Simple test for the Sigmoid Cross Entropy Layer.
+ */
+
+BOOST_AUTO_TEST_CASE(SimpleSigmoidCrossEntropyLayerTest)
+{
+  arma::mat input1, input2, output, target1, target2, expectedOutput;
+  SigmoidCrossEntropyError<> module;
+
+  // Test the Forward function on a user generator input and compare it against
+  // the manually calculated result.
+  input1 = arma::mat("0.5 0.5 0.5 0.5 0.5 0.5 0.5 0.5");
```

Comment:
Since the behavior is different depending on whether the input is positive or negative, should you add some tests with negative inputs?